root/frontend/src/components/Weather/Weather.style.tsx

```diff
@@ -0,0 +1,25 @@
+import { createUseStyles } from 'react-jss';
+
+const useStyles = createUseStyles({
```

Comment:
I noticed that there are some `px` sizings noted here, have you tried these with variable viewport sizes? Just keeping a note on possible scaling issues that may pop up that could lead to wonky visuals on different devices.